users/relaying.rst

```diff
@@ -38,6 +61,10 @@ smooth operation::
     Usage of relaysrv:
       -debug
             Enable debug output
+      -ext-address string
+            An optional address to advertising as being available on.
+            Allows listening on an unprivileged port with port forwarding from e.g.                                             
+            443, and be connected to on port 443.
```

Comment:
I'm not sure it's worth talking about an external IP. By default, the relay doesn't specify the IP it's located at, and the relay pool server looks at where the relay's registration request came from. 

How about...

An optional port that is the real external port of the relay server, e.g. a port which is forwarded by NAT. Use this and a firewall rule to have an unprivileged relaysrv listen on a privileged port.